apps/3000-home/pages/dynamic-remote.js

```diff
@@ -0,0 +1,43 @@
+import React from 'react'
+import {injectScript} from "@module-federation/utilities";
+// example of dynamic remote import on server and client
+const isServer = typeof window === 'undefined';
+//could also use
+// getModule({
+//   remoteContainer: {
+//     global: 'app2',
+//     url: 'http://localhost:3002/remoteEntry.js',
+//   },
+//   modulePath: './sample'
+// }).then((sample) => {
+//   console.log(sample)
+// });
+const dynamicContainer = injectScript({
+  global: 'checkout',
+  url: `http://localhost:3002/_next/static/${isServer ? 'ssr' : 'chunks'}/remoteEntry.js`,
+}).then((container) => {
+  return container.get('./CheckoutTitle').then((factory) => {
+    return factory()
+  })
+})
+const DynamicComponent = React.lazy(() => dynamicContainer);
+
+export default (props)=> {
+  return (
+    <>
+    <React.Suspense>
+      <DynamicComponent/>
```

Comment:
question: will DynamicComponent be rendered in SSR?

Reply:
yes, all my tools are 100% isomorphic